schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name TEXT NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT NOT NULL,
+    neutered BOOLEAN NOT NULL,
+    weight_kg DECIMAL(10,2),
+    PRIMARY KEY(id)
```

Comment:
- [ ] [REQUIRED] Good job so far! I would be nice if you add `name varchar(255)` option to `animals` table 👍🏼 `[NOTE]` That `varchar` columns can have different length specified.
```sql
    CREATE TABLE animals (
    id int,
    name varchar(255),
    date_of_birth date,
    escape_attempts int,
    neutered boolean,
    weight_kg decimal, 
    );
```